addons/website_sale/views/templates.xml

```diff
@@ -267,6 +267,20 @@
                                 t-att-title="ptav.name"
                             />
                         </div>
+                        <div
+                            t-elif="ptav.display_type == 'image'"
+                            class="d-flex"
+                        >
+                            <div
+                                class="css_attribute_image size_small cursor-pointer d-block rounded bg-body"
+                                t-att-title="ptav.name"
+                            >
+                                <div
+                                    class="oe_img_bg o_bg_img_center w-100 h-100 rounded-1"
+                                    t-att-style="'background-image:url(/web/image/product.attribute.value/%s/image);' % ptav.product_attribute_value_id.id"
```

Comment:
This will show the thumbnail image on the `/shop` if no image is given, where the product page will just show a custom CSS placeholder. I'd like both pages to show the information in the same way, whether using the backend placeholder or the CSS one.
```suggestion
                                    t-att-style="'background-image:url(/web/image/product.attribute.value/%s/image); background-size:cover;' % v.id if v.image else ''"
```